addons/hr_skills/views/hr_employee_cv_templates.xml

```diff
@@ -106,21 +102,20 @@
         </div>
 
         <t t-set="has_skills" t-value="show_skills and any(skill_line.skill_type_id != skill_type_language for skill_line in o.employee_skill_ids)"/>
-        <div class="o_main_panel mt-4" t-if="has_skills">
+        <div class="o_main_panel w-100 mt-4 px-3 pt-1 pb-0 align-top" t-if="has_skills">
             <div class="o_main_panel_section">
-                <h2 class="o_main_panel_title" t-attf-style="color: #{color_secondary};">
-                    <span class="o_main_panel_icon" t-attf-style="background: #{color_secondary};">
-                        <i class="fa fa-solid fa-rocket"/>
-                    </span>Skills
+                <h2 class="o_main_panel_title position-relative mt-0 mb-3 fs-5 fw-bold text-uppercase" t-attf-style="color: #{color_secondary};">
+                        <i class="fa fa-solid fa-rocket fa-stack me-2 rounded-circle fs-6 text-white text-center" t-attf-style="background-color: #{color_secondary};"/>
+                    Skills
                 </h2>
                 <t t-set="valid_skills" t-value="[skill for skill in o.employee_skill_ids if skill.skill_type_id != skill_type_language]"/>
                 <!-- Thanks wkhtmltopdf -->
                 <t t-foreach="[valid_skills[n:n+2] for n in range(0, len(valid_skills), 2)]" t-as="skills_pair">
                     <div class="o_main_panel_skills row ps-5">
                         <t t-foreach="skills_pair" t-as="skill_line">
                             <div class="o_main_panel_skill_line col-6 pe-4" t-if="skill_line.skill_type_id != skill_type_language">
-                                <h3 class="o_main_panel_skill_name" t-field="skill_line.skill_id"/>
-                                <div class="progress o_main_panel_skill_progress_bar">
+                                <h3 class="o_main_panel_skill_name mt-0 mb-2 fs-6 fw-bolder" t-field="skill_line.skill_id"/>
+                                <div class="progress o_main_panel_skill_progress_bar rounded-1 bg-200" style="height: 12px;">
```

Comment:
Why define inline style ? 🤔

Reply:
Inline-style is the default styling for BS's progress bar https://getbootstrap.com/docs/5.1/components/progress/#height

It could also be defined in the SCSS file but the result would be the same (unless we prefer using external style for that kind of component)

Reply:
Even though Bootstraps uses inline css, we try to avoid this as much as possible 🙂  
Also, I don't think this custo is necessary since the default height of the progress bar is set to `1em`. Can you check ?